src/components/Common/AccordionWrapper.js

```diff
@@ -18,6 +18,7 @@ const AccordionWrapper = ({
   defaultShowHelp,
   helpButtonText,
   helpText,
+  horizontal = false,
```

Comment:
What is this used for? I don't see a reference for it in this file.

In the future, please use `defaultProps` to set the default value of props passed to components: https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values